source/disruption/line_reports_api.h

```diff
@@ -29,8 +29,7 @@ www.navitia.io
 */
 
 #pragma once
-#include "type/type.h"
```

Comment:
kill the type.h :crossed_swords:

Reply:
![kill it](https://i.giphy.com/media/13nDaxXRXz0Prq/giphy.webp)